src/test-fn/kotlin/com/anaplan/engineering/vdmgradleplugin/CompositeTestTest.kt

```diff
@@ -39,7 +39,6 @@ class CompositeTestTest(
         @Parameterized.Parameters(name = "{0}")
         fun example() = arrayOf(
                 test(testName = "compositePassingTest"),
-                test(testName = "compositeWithSharedPassingTest"),
```

Comment:
I couldn't find a straightforward way to share the dependency declaration in the `buildscript` block.